scripts/python/docs-automation/sync_tocs.py

```diff
@@ -202,15 +202,15 @@ def grep_children_namespaces(autogenerated_toc_yml):
             target_autogenerated_toc = yaml.safe_load(target_autogenerated_toc_yml)
     except Exception as f:
         print(
-            "Execution requires that both the known namespaces and reference yml be defined."
+            "Execution requires that both the known namespaces and reference yml be defined. Error message: " + str(f)
         )
 
     present_in_target = grep_children_namespaces(target_autogenerated_toc)
 
     print(
         "Here are the visible namespaces in target autogenerated ToC. Constraining reference.yml."
     )
-    for ns in sorted(present_in_target):
+    for ns in sorted(list(dict.fromkeys(present_in_target))):
```

Comment:
❤️

Reply:
> > What's causing these duplications?
> 
> What we do is only read the toc.yml generated from Docs site tool.
> 
> ```
> def grep_children_namespaces(autogenerated_toc_yml):
>     return [
>         top_level_namespace["name"] for top_level_namespace in autogenerated_toc_yml
>     ] + ["*"]
> ```
> 
> We have a step for `Merge Un-Migratable Legacy Packages To Stable ToC` which read APIs from [migration.json](https://github.com/Azure/azure-docs-sdk-java/blob/main/temporary-src-based-yml/migration.json)
> 
> The duplication APIs from all from this migration.json file.

Have you cleaned any packages that are in `latest` now out of the `migration.json`? If there is duplication that indicates to me that we are _overwriting_ the `latest` bits with output from the migration.json file.